recipes/cookiecutter-django/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "cookiecutter-django"%}
```

Comment:
I'm not that familiar with the cookiecutter project, but it looks like this isn't a package -- instead it's just a repo template. I'm not sure how this would be used in the context of conda. I realize that there is a pypi package, but none of the usage descriptions refer to pip installing it. Can you describe how you would use this with conda?

Reply:
We want to be able to ship a versioned release of this repo to users via conda forge so that users can use conda to install a specific version of cookiecutter-django along with all of its dependencies.  In other words, we want to package the repo and use conda forge for dependency management and installation.  Then the user can run cookiecutter and point it at the installation folder and they'll have all the dependencies installed already.

Reply:
So `cookiecutter-dango` becomes the dependency of the repo created via the template?

Reply:
So let me try to walk you through the target workflow:
* `conda install cookiecutter-django` : this command will install cookiecutter-django, cookiecutter, django, and all the rest of the listed requirements in cookiecutter-django.
* `cookiecutter /path/to/cookiecutter-django` : this command will create a cookiecutter django project using the cookiecutter-django template.  

All of the dependencies that the newly created django project requires will have already been installed by the `conda install` command.  Does that make sense?

Reply:
@cshaley I really like this idea. Can we agree on a common path to where cookiecutter templates get installed under $CONDA_PREFIX?

Reply:
Alternatively, do you think we could setup .cookiecutterc with the conda environment such that we could render conda installed templates using `cookiecutter conda:<template-name>`? For this case in particular, `cookiecutter conda:django`.